### ISSUE TITLE
Marriage full date is not consistent

### ISSUE THREAD

**Issue by hartenthaler** [magicsunday/webtrees-fan-chart#65]
Great that it is now possible to show the marriage dates, too! But you show them with day and month and not only with year as all the other dates. Maybe it is possible to show birth and death dates with day and month, too. At least in the generations 0 to 3.

**Comment by reteP-riS** [comment]
I agree with that proposal and suggest to use * (`&ast;`) and † (`&dagger;`) for birth and death dates.
